rigby/rigby/nodes/gr24/gps.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+from .data_node import DataNode
+
+'''
+Latitude				High Precision Latitude			
+Longitude				High Precision Longitude			
+
+'''
+
+class GPS(DataNode):
+  latitude : int
+  high_precision_latitude : int
```

Comment:
ignore the high precision fields just assume we are going to have latitude and longitude, each taking up 8 bytes (64 bit floats)